apps/plato_single_variant/dxapp.json

```diff
@@ -11,124 +12,143 @@
       "name": "input_plink_binary",
       "label": "PLINK binary input files (bed, bim, fam)",
       "class": "array:file",
-      "optional": false
+      "patterns": ["*.bed", "*.bim", "*.fam"],
+      "optional": false,
+      "help": "A list of files with PLINK's binary fileset to be referenced."
     },
     {
       "name": "input_phenotype",
```

Comment:
update variable names with suggestions from the last meeting, e.g. phenotypes_tsv